src/praisonai/tests/unit/bots/test_store_backed_identity.py

```
@@ -0,0 +1,133 @@
+
+def test_unlink_reverts_to_fallback(resolver):
+    resolver.link("telegram", "12345", "alice")
+    resolver.unlink("telegram", "12345")
+    assert resolver.resolve("telegram", "12345") == "telegram:12345"
+
+
+def test_links_for_lists_all_channels(resolver):
+    resolver.link("telegram", "12345", "alice")
+    resolver.link("whatsapp", "+44123", "alice")
+    links = resolver.links_for("alice")
+    pairs = {(l.platform, l.platform_user_id) for l in links}
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)